samza-core/src/main/java/org/apache/samza/task/AsyncRunLoop.java

```diff
@@ -91,6 +94,8 @@ public AsyncRunLoop(Map<TaskName, TaskInstance> taskInstances,
     this.threadPool = threadPool;
     this.consumerMultiplexer = consumerMultiplexer;
     this.containerMetrics = containerMetrics;
+    this.containerMetricsCompletedCounter = containerMetrics.newCounter("async-callback-complete-calls");
```

Comment:
Please follow the pattern we had to declare/add metric in SamzaContainerMetrics (i.e. instantiate the metric inside SamzaContainerMetrics class and just refer to it from outside like containerMetrics.callbackCompletes.inc()

Reply:
Done